### PR TITLE
#4 - General Styles

### LINKED ISSUE #4: Add general styles to the site
- Identify breakpoints for different sizes (we'll want to make mobile the default)
- Add default font sizes and font files (both for the body and different heading elements)
- Add default inter-section margins
- See if there's anything else worth reusing

Some pages worth looking at:
- https://www.esri.com/en-us/about/about-esri/europe/overview
- https://www.esri.com/en-us/capabilities/imagery-remote-sensing/capabilities/management
- https://www.esri.com/en-us/digital-transformation/overview

### PR DESCRIPTION
**Pull request by stefanalex99**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #4

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri--aemsites.hlx.live/en-us/about/about-esri/overview
- After: https://4-general-styles--esri--aemsites.hlx.live/en-us/about/about-esri/overview

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -20,44 +20,43 @@
...
   --heading-font-family: var(--body-font-family);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
 
-  /* body sizes */
-  --body-font-size-m: 22px;
-  --body-font-size-s: 18px;
-  --body-font-size-xs: 16px;
-
-  /* heading sizes */
-  --heading-font-size-xxl: 48px;
```
Comment (alexcarol) [inline styles/styles.css:33]: These variables are important because they're used here: https://github.com/aemsites/esri/blob/d54f2255cab85b4168080f9fbe5bc2ad5caea1d7/styles/styles.css#L93

What do all these "font--<number>" mean?

File: styles/styles.css [hunk 2]
```diff
@@ -36,37 +56,12 @@
...
-    --heading-font-size-l: 36px;
-    --heading-font-size-m: 30px;
-    --heading-font-size-s: 24px;
-    --heading-font-size-xs: 22px;
-  }
-}
 
 body {
-  font-size: var(--body-font-size-m);
-  margin: 0;
```
Comment (alexcarol) [inline styles/styles.css:64]: Why did you remove this? From what I see the margin seems to be 0 in most pages, do you have any counterexamples?

File: styles/styles.css [hunk 3]
```diff
@@ -36,37 +56,12 @@
...
-  font-family: var(--body-font-family);
```
Comment (alexcarol) [inline styles/styles.css:65]: Isn't this needed? Right now the body has no specific font-family

File: styles/styles.css (hunk 1 above)
Comment (tim-white-esri) [reply styles/styles.css:33]: Yes, we have a variable font scale set up based on that scale in `:root`

**Comment by tim-white-esri** [comment]
I have a full font file that we can talk about. We can either point to it or I can recreate it on this side. 
Our current site uses 
<link rel="preload" as="style"
	href="https://fast.fonts.net/t/1.css?apiType=css&projectid=f9e335c8-e150-4885-976e-f4c392e562c4">
<link rel="preload" as="style" href="https://webapps-cdn.esri.com/CDN/fonts/v1.4.2/fonts.css">

**Comment by tim-white-esri** [comment]
@stefanalex99 I have our full font file that I can include, along with all of the proper woff2 files. 
I can create a new issue and update the font file

**Review by alexcarol — APPROVED** [review]
